Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
```

Comment:
Alright.

`:revision` is necessary because:

* It gives us extra verification and some security usefulness. It is
essentially a checksum for that tag, verifying the tag matches the
expected hash.
* Tags can be changed. Commits are unique.